### PR TITLE
Image: Consolidate crop ratios

### LINKED ISSUE #22567: Consolidate portrait and landscape image editing aspect ratios
Follow-up from https://github.com/WordPress/gutenberg/pull/21024#issuecomment-632567310

Having a separate button to switch between portrait and landscape may not be intuitive. Instead, it might make more sense to have the portrait and landscape options both listed in the aspect dropdown.

aspect dropdown

### PR DESCRIPTION
**Pull request by ajlende**
## Description

This removes the portrait/landscape button in favor of a simpler, unified aspect ratio menu with both portrait and landscape ratios in the same menu.

## How has this been tested?

Add image block, open crop, view the unified aspect ratio menu.

## Screenshots 

image

## Types of changes

Fixes #22567 

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-library/src/rich-image/editor.scss [hunk 1]
```diff
@@ -85,3 +85,13 @@
 .richimage-toolbar__control {
 	padding: 6px;
 }
+
+.richimage__aspect-title {
```
Comment (ellatrix) [inline packages/block-library/src/rich-image/editor.scss:89]: Is there no component for this? I see a similar thing in the settings:

Screenshot 2020-06-06 at 17 05 41

File: packages/block-library/src/rich-image/rich-image/index.js [hunk 2]
```diff
@@ -288,52 +288,111 @@ class RichImage extends Component {
...
-								label={ __( 'Orientation' ) }
-								onClick={ () =>
-									this.setState( ( prev ) => ( {
-										isPortrait: ! prev.isPortrait,
-									} ) )
-								}
-							/>
+							>
+								{ ( { onClose } ) => (
+									<Fragment>
```
Comment (ellatrix) [inline packages/block-library/src/rich-image/rich-image/index.js:293]: Minor: I think we prefer `<>`, but if there's no linter rule for it... 🤷‍♀️

File: packages/block-library/src/rich-image/editor.scss (hunk 1 above)
Comment (ajlende) [reply packages/block-library/src/rich-image/editor.scss:89]: That's where it is! I though I had seen it somewhere, but I couldn't remember where and I couldn't find a component for it, so I assumed I had imagined it. Looks like it's still just a div, but at least I can grab the class name from it.

**Review by ellatrix — APPROVED** [review]

**Comment by ellatrix** [comment]
Just needs a rebase now :)

**Pull request merged**